metadata-ingestion/src/datahub/ingestion/source/dbt.py

```diff
@@ -213,11 +213,12 @@ def aws_connection_needed_if_s3_uris_present(
         cls, aws_connection: Optional[AwsConnectionConfig], values: Dict, **kwargs: Any
     ) -> Optional[AwsConnectionConfig]:
         # first check if there are fields that contain s3 uris
-        uri_containing_fields = [
-            f
-            for f in ["manifest_path", "catalog_path", "sources_path"]
-            if values.get(f, "").startswith("s3://")
-        ]
+        uri_containing_fields = []
+        for f in ["manifest_path", "catalog_path", "sources_path"]:
```

Comment:
Hi @BoyuanZhangDE, could you please retain the original list comprehension style? You just need to replace
```python
            if values.get(f, "").startswith("s3://")
```
with 
```python
            if (values.get(f) or "").startswith("s3://")

```